src/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+import org.junit.Test;
+
+public class NameTest {
+	@Test
+	public void testNothing() {
+		
```

Comment:
I think this method doesn't do anything right? May be instead you can try calling the `isSimilar` method here, before writing the method.